client/src/lib/repositories/tags.ts

```diff
@@ -0,0 +1,17 @@
+import type { Fetch } from "src/definitions/fetch";
+import type { Tag } from "src/definitions/tag";
+import { getApiUrl, getHeaders, makeApiRequest } from "../fetch";
+import { Maybe } from "../util/maybe";
+
+type GetTags = (opts: {
+  fetch: Fetch;
+  apiToken: string;
+}) => Promise<Maybe<Tag[]>>;
+export const getTags: GetTags = async ({ fetch, apiToken }) => {
+  const url = `${getApiUrl()}/tags`;
```

Comment:
Inclure un `/` final sinon l'API effectue un `307 Temporary Redirect` à chaque appel

```console
[server] INFO:     127.0.0.1:33888 - "OPTIONS /tags HTTP/1.1" 200 OK
[server] INFO:     127.0.0.1:33888 - "GET /tags HTTP/1.1" 307 Temporary Redirect
[server] INFO:     127.0.0.1:33888 - "OPTIONS /tags/ HTTP/1.1" 200 OK
[server] INFO:     127.0.0.1:33888 - "GET /tags/ HTTP/1.1" 200 OK
```


```suggestion
  const url = `${getApiUrl()}/tags/`;
```